libraries/helpers.rb

```diff
@@ -113,8 +113,6 @@ def product_lookup(product, version = '0.0.0')
       elsif (product == 'push-server') && (v < Mixlib::Versioning.parse('2.0.0'))
         data['package-name'] = 'opscode-push-jobs-server'
         data['ctl-command'] = 'opscode-push-jobs-server-ctl'
-      elsif (product == 'push-client') && (v < Mixlib::Versioning.parse('2.0.0'))
```

Comment:
Ignore that -- there may be more to this -- I didn't see it in my `apt-cache policy` output before -- the packages do exist on package cloud, however it may not be available for all versions of Ubuntu. In my case, 14.04 doesn't return a match.

```
$ apt-cache policy opscode-push-jobs-client
opscode-push-jobs-client:
  Installed: (none)
  Candidate: (none)
  Version table:
$ apt-cache policy push-jobs-client
push-jobs-client:
  Installed: (none)
  Candidate: 1.3.4-1
  Version table:
     1.3.4-1 0
        500 https://packagecloud.io/chef/stable/ubuntu/ trusty/main amd64 Packages
     1.3.3-1 0
        500 https://packagecloud.io/chef/stable/ubuntu/ trusty/main amd64 Packages
```